docs/source/starthere/fundamentals.rst

```diff
@@ -0,0 +1,240 @@
+NeMo fundamentals
+=================
+
+On this page we will go a little bit deeper on how NeMo works, to give you a good foundation for using NeMo for your :ref:`desired usecase <where_next>`.
+
+NeMo Models
+-----------
+
+A NeMo "model" includes all of the below components wrapped into a singular, cohesive unit:
+
+* neural network architecture,
+
+* dataset & data loaders,
+
+* data preprocessing & postprocessing,
```

Comment:
what's postprocessing of data?

Reply:
stuff like decoding strategies (greedy, beam etc.) and turning output logprobs into strings of responses
e.g. for ASR the "decoding" class does all this stuff

Reply:
will correct to "postprocessing of model outputs"